sdk/servicebus/azure-messaging-servicebus/src/test/java/com/azure/messaging/servicebus/ServiceBusReceiverAsyncClientIntegrationTest.java

```diff
@@ -63,6 +62,7 @@
 
     private ServiceBusReceiverAsyncClient receiver;
     private ServiceBusSenderAsyncClient sender;
+    private ServiceBusSessionReceiverAsyncClient sessionReceiver;
```

Comment:
Do we really need this to be a member variable? Isn't this only tested in a few test cases.

Reply:
It is used for any session based test, we have atleaset 8-10 of them.
The session test, user receiver and that receiver is created using this `ServiceBusSessionReceiverAsyncClient` and we have to close it.